src/jira/client/jira-client.ts

```diff
@@ -206,11 +206,7 @@ export const getJiraClient = async (
 						!withinIssueKeyLimit(data.branches) ||
 						!withinIssueKeyLimit(data.pullRequests)
 					) {
-						logger.warn({
```

Comment:
@bgvozdev - This is just a massive list of issue keys, I assumed issue keys fall under UGC but would like a confirm?